### PR TITLE
Create static YouTube clone 

### PR DESCRIPTION
**Pull request by GutemaG**
# Create YouTube clone
- Layout the structure of YouTube
- Download YouTube logo
- Add README with showcase

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# code-review-for-youtube-html-css-clone
+# microverse-youtube-html-css-clone
+Microverse Pre-Enrollment Bootcamp html-css task 
+
+![alt text](https://github.com/GutemaG/microverse-youtube-html-css-clone/blob/master/images/screen-1.png)
+![alt txt](https://github.com/GutemaG/microverse-youtube-html-css-clone/blob/master/images/screen-2.png)
```
Comment (GutemaG) [inline README.md:5]: - It is very interesting that you used image to show your project

- Microverse provide an amazing README template, so it is very good that you use their template

**Comment by GutemaG** [comment]
## Wow, almost there
- Good usage of GitHub flow
- Descriptive Message for Review message

## It is better to change the following:
- It is good to use GitHub action (CI/CD). Try to use HTML&CSS linter.
- It is very good practice if you apply html and CSS best practice.
- For readability, use two space indentation.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+            </form>
+        </div>
+        <!-- more icon -->
+        <div>
+            <i class="material-icons">
+                more
+            </i>
+            <i class="material-icons ">account_circle</i>
+        </div>
+    </div>
```
Comment (GutemaG) [inline index.html:43]: ### It is good practice if you use header and nav semantic html tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+    <!-- Mani Body ...start-->
+    <div class="main">
+        <!-- <div class="sidebar">
+            <h1>
+                Side Bar
+            </h1>
+        </div> -->
+
+        <!-- End of Sidebar -->
+
```
Comment (GutemaG) [inline index.html:53]: ## For readability, avoid long comments

File: index.html [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+        <!-- more icon -->
+        <div>
+            <i class="material-icons">
+                more
+            </i>
+            <i class="material-icons ">account_circle</i>
+        </div>
+    </div>
+    <!-- Mani Body ...start-->
+    <div class="main">
```
Comment (GutemaG) [inline index.html:45]: I think it would be better to use the section semantic tags

File: README.md (hunk 1 above)
Comment (karlavdelgadof) [reply README.md:5]: **[OPTIONAL]:** Awesome job pointing out what the student did well! :partying_face: , however, for your comment I would suggest rephrasing it as a suggestion or recommendation like so:

- "I think you could..." :bulb: 
- "I would suggest..." :+1: 

And after you could provide an explanation or possible benefit of introducing this change. In this way, you can make sure that the student gets a clear idea of the requested change :100:

**Review by GutemaG — COMMENTED** [review]
## Wow, almost there
- Good usage of GitHub flow 👍 
- Descriptive Message for Review message

## It is better to change the following:
- It is good to use GitHub action (CI/CD). Try to use HTML&CSS linter.
- It is very good practice if you apply html and CSS best practice.
- For readability, use two space indentation.

**Review by karlavdelgadof — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi there @GutemaG :smile: :wave: !

You have successfully completed this project! There is nothing else to say other than... it's time to merge it! 🚀
Time to celebrate, you have done a fantastic job, congratulations! 🎉



## :sparkles: Highlights :sparkles:


- Great work using kind and precise language to communicate with the student :100: :clap: 

APPROVED! 💯 💯 💯

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Cheers and best of luck with the following projects! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@karlavdelgadof**  in your question so I can receive the notification.
  
  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
  
  ------
  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**